apps/web/src/lang/fr.json

```diff
@@ -7978,6 +8762,10 @@
     "defaultMessage": "<strong>Préavis!</strong> Avant de vous demander d’aller établir vos justificatifs CléGC, nous vous demandons d’examiner de nouveau les critères d’admissibilité du programme afin de veiller à ce que vous répondiez aux exigences.",
     "description": "A request for the user to review the IAP eligibility criteria"
   },
+  "ozKHDz": {
+    "defaultMessage": "Si plusieurs types de ressources ont été établis sous {nom de la section}, veuillez inclure un numéro de suivi pour chaque ressource qui relève de volets de travail de la TI et/ou d’un niveau distinct. Séparez chaque numéro de suivi par une virgule.",
```

Comment:
Variable name was translated. 
Appears to have been causing build issues too, naturally.

Reply:
Good catch. Fixed in [french fixes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7771/commits/407711cf8e2fb91a852997ee706d369a42597e29).